installer/conf/rsyslog-lad.conf

```diff
@@ -0,0 +1,2 @@
+# OMS Syslog collection for workspace LAD (SYSLOG_PORT=%SYSLOG_PORT%)
```

Comment:
I'd prefer not to, because removing "OMS" will break the UnconfigureRsyslog() function at [here](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/scripts/configure_syslog.sh#L72). All the code changes here were deliberately chosen to make them work with the existing configure_syslog.sh and/or omsadmin.sh logic, without disturbing them too much.